profilegen/src/res/templates/module_template.template

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+require '<%= @module_require %>'
+
+module Azure
+  module Management
+    module Profiles
+      module <%= @profile_name %>
+        module <%= @module_name %>
+          #
+          # Class for the <%= @class_name %> module
+          class <%= @class_name %>
+            attr_accessor <% @operation_types.each do |operation_type| %> :<%= operation_type[:operation_name] %>,<% end %> :model_types
+
+            def initialize(configurable, base_url, options)
+              client = <%= @management_client %>.new(configurable.credentials, base_url, options)
+              client.subscription_id = configurable.subscription_id
+              <%- @operation_types.each do |operation_type| -%>
+              self.<%= operation_type[:operation_name] %> = <%= operation_type[:operation_body] %>.new(client)
```

Comment:
As discussed, did you find out whether we really need to new the each operation again? That may lead to performance hit and memory usage

Reply:
Actually Yes. Because, each may point to a different version of RT

Reply:
can we elaborate a bit more on this, I remember a bit of both discussions, but want to make sure this is really needed.